clinvar_ingest/api/main.py

```diff
@@ -59,18 +63,24 @@ async def copy(payload: ClinvarFTPWatcherRequest):
             ftp_path=ftp_path,
             gcs_path=gcs_path,
         )
-    except Exception:
-        msg = f"Failed to copy {ftp_path}."
+    except Exception as e:
+        msg = f"Failed to copy {ftp_path}"
         logger.exception(msg)
         raise HTTPException(
             status_code=status.HTTP_422_UNPROCESSABLE_ENTITY,
             detail=msg,
-        )
+        ) from e
 
 
-@app.post("/parse", status_code=status.HTTP_201_CREATED)
+@app.post("/parse", status_code=status.HTTP_201_CREATED, response_model=ParseResponse)
 async def parse(payload: ParseRequest):
-    return {"todo": "implement me"}
+    output_files = parse_and_write_files(
+        payload.input_path,
+        payload.output_path,
+        disassemble=not payload.no_disassemble,
```

Comment:
I think we should only have one copy of these... just to reduce entropy.  So either pass `no_disassemble` forwards or change the `ParseRequest` to take `disassemble`.

Reply:
Agreed, it's getting confusing with the double negatives

Reply:
https://github.com/clingen-data-model/clinvar-ingest/issues/69